src/git-metadata/api.ts

```diff
@@ -0,0 +1,11 @@
+export const datadogSite = process.env.DATADOG_SITE || "datadoghq.com";
```

Comment:
If I understand correctly, this means that the DATADOG_SITE must be set using an environment, variable and we don't use the `site` parameter from the config file. What is the rationale for that?

If we do have a separate environment variable/setting for this, we probably need to document that, and also validate that it's one of the valid values (e.g. not misspelled).

Reply:
Good catch, this was just leftover cruft from datadog-ci -- I'll use the `site` that we use in serverless.